### DISCUSSION TITLE
Custom layouts debugging

### DISCUSSION THREAD

**Discussion by melta** [ianyh/Amethyst#1441] [General]
Hi guys,

Is there a way to debug custom layouts? I have a few of them and somtimes a wierd things happen when I switch them back and forth. It would be great to be able to print some debug information to console or something like that. Can anyone help me with that. Or point me to the right direction. Thanks in advance.

**Comment by ianyh** [comment]
You can see this issue for more details, but logging to console from the layout should appear in the macOS console, as visible from the Console app. https://github.com/ianyh/Amethyst/issues/1294

**Reply by melta** [reply]
> You can see this issue for more details, but logging to console from the layout should appear in the macOS console, as visible from the Console app. #1294

Thank you very much @ianyh !
